roles/systems/postgres/tasks/main.yaml

```diff
@@ -101,7 +101,7 @@
     - name: Copy Ubuntu default SSL Cert
       ansible.builtin.copy:
         src: /etc/ssl/certs/ssl-cert-snakeoil.pem
-        dest: /etc/ssl/certs/ssl-cert-sas.pem
+        dest: "/etc/ssl/certs/ssl-cert-sas-{{ ansible_hostname }}.pem"
```

Comment:
Is ansible_hostname set somewhere previously?

Reply:
This is pulled from the ansible facts gathers by each hosts. Here's the [link](https://docs.ansible.com/ansible/latest/user_guide/playbooks_vars_facts.html) to the doc.

Reply:
I had to take a look at the Ansible doc to verify why this worked.
It looks like `ansible_hostname` gets set for each host when facts are gathered
https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html#connection-variables

In the project, `gathering` is set to smart
https://github.com/sassoftware/viya4-iac-k8s/blob/multi_pg/ansible.cfg#L9
Which means (from the doc)
> The value ‘smart’ means each new host that has no facts discovered will be scanned, but if the same host is addressed in multiple plays it will not be contacted again in the playbook run.
https://docs.ansible.com/ansible/2.4/intro_configuration.html#id111

So whatever the first task to run on the PG machine was, set the `ansible_hostname` variable